Microsoft.Toolkit.Win32/Microsoft.Windows.Interop.WindowsXamlHost/IDesktopWindowXamlSourceNative.cs

```diff
@@ -0,0 +1,58 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Windows.Interop
```

Comment:
Should be in a different namespace?

Reply:
Code in this Microsoft.Windows.Interop namespace was code that was provided to me...  I'm not sure why StyleCop's not' picking it up, but fixing all the issues it might pick up could take some time.

Reply:
I've fixed this up in 758d2f080d6d8aeb50892b5fd97cc3daf9c19ad4